Move to scram tool xml

nsmith- opened the pull request:
The only advantage this brings is that `cmsenv` is sufficient to
set up the environment after the initial setup.

Maybe we can also use the version syntax to keep track?
I haven't found any way to make a BuildFile download things through svn yet, but I'm sure there's a way...

nsmith- committed:
Move to scram tool xml

The only advantage this brings is that `cmsenv` is sufficient to
set up the environment after the initial setup.

SridharaDasu commented:
I think I did not think through earlier properly. The temporary measure of having dependence on my public area is bad.  Therefore, I add "make install" to get the shared images copied to appropriate area of CMSSW.  

You seem to understand the scram details much better than me.  Can you now use the instructions from what I have again and see if you can use the "local" version of UCT stuff in $CMSSW_RELEASE to build the Spy?  It seems to be possible.

Thanks.

nsmith- committed:
Output file option

nsmith- committed:
Adding input source, unfinished

nsmith- committed:
Update instructions for scram xml

Staying clear of $CMSSW_BASE for fear that scram might do something to
extern that we don't expect

nsmith- committed:
Ironed-out instructions

Installed in /data/rctcmstr/ on uwlogin

nsmith- commented:
Ok, working instructions with the xml stuff
I tried it out on the `rctcmstr` account
